### ISSUE TITLE
Policy routing not working properly

### ISSUE THREAD

**Issue by TalalMash** [SmoothWAN/SmoothWAN#124]
**Fix by typing in terminal:** 
```
sed -i "s+config interface 'Speedify'+config interface 'wan'+" /etc/config/network ; uci add_list firewall.cfg03dc81.network='wan' ; uci commit firewall ; reboot
```

To fix in v1.
Credits to @bt61 for testing, apologies for data capped users.

_Notes: main VPN should be named: `wan` for newer PBR versions._

**Comment by TalalMash** [comment]
https://docs.openwrt.melmac.net/pbr/
![image](https://user-images.githubusercontent.com/96490382/222782023-db8a3b05-b827-492e-a27b-9c5996a80b04.png)

**Comment by TalalMash** [comment]
Add network names (not interface) case-sensitive to `Advanced Configuration`
Example
![image](https://user-images.githubusercontent.com/96490382/222874155-d8af28e4-eb29-46eb-a5a6-fd4c6b0bb886.png)

**Comment by TalalMash** [comment]
Fixed in `0.99.9HF4`
